basics.js

```diff
@@ -31,6 +37,16 @@ def greet(name, excited)
 end
 */
 
+var greet = function greeting (name, excited) {
+ var agreeting = "Hi, my name is #{name}";
+ if (excited) {
+  console.log(agreeting.upcase());
```

Comment:
I see what you're thinking, but .upcase isn't a method in JS, and if it was, using console.log makes it so that it will return undefined (like `puts` would do in Ruby)